_data/students.yml

```diff
@@ -101,3 +101,7 @@
   github: saarthakchats
   image: saarthak.JPG
   twitter: SaarthakChats41
+
+- name: Akshit Goyal
+  github: akshit15013
```

Comment:
This image is out of dimensions

Reply:
How much is the image size Of my image?

Reply:
I HAVE A MODIFIED CODE , CAN YOU PLEASE MERGE IT?